packages/integration-tests/tsconfig.json

```diff
@@ -9,13 +9,12 @@
         ],
         "noEmit": true
     },
-    "include": ["**/*.ts", "**/*.js"],
+    "include": [
+        "**/*.ts",
+        "**/*.js",
+        "../suite-web/e2e/process-har.ts"
```

Comment:
what is this `integration-tests` package good for now?

Reply:
I dont know yet, at the moment, there are still some tests left (connect-webextension) that are not part of this PR. In the future I am going to either remove it, or turn it into some kind of e2e-common package